hs/src/Reach/Eval/Core.hs

```diff
@@ -1444,7 +1445,8 @@ evalAsEnvM sv@(lvl, obj) = case obj of
   SLV_Type (ST_Bytes len) -> do
     return $ Just $
       M.fromList
-        [("pad", retV $ public $ SLV_Prim $ SLPrim_padTo len)]
+        [ ("pad", retV $ public $ SLV_Prim $ SLPrim_padTo len)
+        , ("fromHex", retV $ public $ SLV_Prim $ SLPrim_Bytes_fromHex len)]
```

Comment:
If you added the number because you thought you couldn't add keys... you can add keys to any slprim